vendor/github.com/cyverse-de/messaging/amqp.go

```diff
@@ -33,12 +33,6 @@ var (
 	//JobsExchange is the name of the exchange that job related info is passed around.
 	JobsExchange = "jobs"
 
-	//ReindexExchange is the name of the exchange that full-reindex info is passed around on.
-	ReindexExchange = "de"
-
-	//IncrementalExchange is the name of the exchange that incremental update info is passed around on.
-	IncrementalExchange = "de"
-
```

Comment:
This way of removing looks incorrect, since I don't see any modification in the messaging repo or the vendor manifest. It's not critical that these be removed in this PR, just that they're removed in the upstream repo since they're not used any more. It looks like CORE-8330 covers that, but I'd not mess with vendored files directly like this.